deploy/base/webhook/rbac.yaml

```diff
@@ -75,3 +75,30 @@ roleRef:
   kind: Role
   name: leader-elector
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: spo-log-enricher-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+  - list
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: spo-log-enricher-binding
+  namespace: security-profiles-operator
+subjects:
+- kind: ServiceAccount
+  name: security-profiles-operator
+  namespace: security-profiles-operator
+roleRef:
+  kind: ClusterRole
+  name: spo-log-enricher-role
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Should this actually be in `deploy/base/rbac.yaml`?

Reply:
Thank you for spotting that, it is now fixed.